From cfea8865cba0264777a3cc464ffba64f0102317f Mon Sep 17 00:00:00 2001
From: Anirban Bose <boses156@gmail.com>
Date: Fri, 10 Jul 2026 14:11:39 +0000
Subject: [PATCH 03812/15333] staging: rtl8723bs: remove unused variable
 bDumpRxPkt

The 'bDumpRxPkt' variable is declared validate_recv_frame() function and
some debug information is written into it by rtw_hal_get_def_var() but
as I have checked it, it is not used anywhere else in the function so
its basically a useless variable now.

I have remove the variable and its assignment in the function.

Signed-off-by: Anirban Bose <boses156@gmail.com>
Link: https://patch.msgid.link/20260710141139.20541-1-boses156@gmail.com
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/rtl8723bs/core/rtw_recv.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/staging/rtl8723bs/core/rtw_recv.c b/drivers/staging/rtl8723bs/core/rtw_recv.c
index 68e26fb139c3..16ec2248b083 100644
--- a/drivers/staging/rtl8723bs/core/rtw_recv.c
+++ b/drivers/staging/rtl8723bs/core/rtw_recv.c
@@ -1448,7 +1448,6 @@ static signed int validate_recv_frame(struct adapter *adapter, union recv_frame
 	u8 type;
 	u8 subtype;
 	signed int retval = _SUCCESS;
-	u8 bDumpRxPkt;
 
 	struct rx_pkt_attrib *pattrib = &precv_frame->u.hdr.attrib;
 
@@ -1474,7 +1473,6 @@ static signed int validate_recv_frame(struct adapter *adapter, union recv_frame
 	pattrib->mdata = GetMData(ptr);
 	pattrib->privacy = GetPrivacy(ptr);
 	pattrib->order = GetOrder(ptr);
-	rtw_hal_get_def_var(adapter, HAL_DEF_DBG_DUMP_RXPKT, &(bDumpRxPkt));
 
 	switch (type) {
 	case WIFI_MGT_TYPE: /* mgnt */
-- 
2.55.0

